src/services/__test__/qr.service.test.ts

```
@@ -0,0 +1,113 @@
+import { UserRepository } from '@/repositories/user.repository';
+import { DBError } from '@/exception';
+import { QRLoginToken } from '@/types/models/QRLoginToken.type';
+
+jest.mock('@/repositories/user.repository');
+
+describe('UserService', () => {
+  let service: UserService;
+  let repo: jest.Mocked<UserRepository>;
+
+  beforeEach(() => {
+    const repoInstance = new UserRepository({} as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)